repo-scripts/size-analysis/analysis-helper.ts

```diff
@@ -731,33 +737,54 @@ export function extractExternalDependencies(
 /**
  * This function generates a binary size report for the given module specified by the moduleLocation argument.
  * @param moduleLocation a path to location of a firebase module
- * @param outputDirectory a path to a directory where the reports will be written under.
- * @param writeFiles when true, will write reports to designated directory specified by outputDirectory.
  */
 export async function generateReportForModule(
-  moduleLocation: string,
-  outputDirectory: string,
-  writeFiles: boolean
-): Promise<string> {
+  moduleLocation: string
+): Promise<Report> {
   const packageJsonPath = `${moduleLocation}/package.json`;
   if (!fs.existsSync(packageJsonPath)) {
-    return;
+    return null;
```

Comment:
We throw if the bundle file doesn't exist. It is the same kind of error, so I think we should throw instead.